specification/resources/resource-manager/Microsoft.Authorization/stable/2020-09-01/examples/getDataPolicyManifest.json

```diff
@@ -0,0 +1,115 @@
+{
+  "parameters": {
+    "api-version": "2020-09-01",
+    "policyMode": "Microsoft.KeyVault.Data"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "id": "/providers/Microsoft.Authorization/dataPolicyManifests/Microsoft.KeyVault.Data",
+        "type": "Microsoft.Authorization/dataPolicyManifests",
+        "name": "Microsoft.KeyVault.Data",
+        "properties": {
+          "namespaces": ["Microsoft.KeyVault"],
+          "policyMode": "Microsoft.KeyVault.Data",
+          "isBuiltInOnly": false,
+          "resourceTypeAliases": [
+            {
+              "resourceType": "Microsoft.KeyVault",
```

Comment:
these are RP/control plane aliases - you don't need to provide so many .. just one data plane alias should be good

Reply:
Thanks. Is resourceType also Microsoft.Keyvault.Data?

Reply:
no, resourceType is vaults/certificates